src/opengl/gpu.c

```diff
@@ -122,6 +122,8 @@ pl_gpu pl_gpu_create_gl(pl_log log, pl_opengl pl_gl, const struct pl_opengl_para
     const char *glslver = (char *) gl->GetString(GL_SHADING_LANGUAGE_VERSION);
     if (glslver) {
         PL_INFO(gpu, "    GL_SHADING_LANGUAGE_VERSION: %s", glslver);
+        if (strncmp(glslver, "OpenGL ES GLSL ES ", 18) == 0)
+            glslver += 18;
```

Comment:
I would prefer to avoid hard-coding the string size, even if it is unlikely to change. I would probably switch to `pl_str` and use `pl_str_eatstart0` here.

Reply:
Changed.